src/main/java/de/szut/lf8_project/lf8/project/dto/ProjectCreateDto.java

```diff
@@ -0,0 +1,73 @@
+package de.szut.lf8_project.lf8.project.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import de.szut.lf8_project.exceptionHandling.InvalidDataException;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.web.client.HttpClientErrorException;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.text.MessageFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Getter
+@Setter
+public class ProjectCreateDto {
+
+    @Min(0)
+    @NotNull
+    private Long customerId;
+
+    @Min(0)
+    @NotNull
+    private Long responsibleEmployeeId;
+
+    @Min(0)
+    @NotNull
+    private Long responsibleCustomerEmployeeId;
+
+    @Size(min = 1)
+    private List<Long> employees;
+
+    @Size(min = 1)
+    private List<String> skills;
```

Comment:
Wir sollten Konsistent bleiben und überall SkillSet verwenden

Reply:
lets not forget our variable naming rules, if we are renaming this variable to SkillSet, It should be "skillSet" and not "SkillSet"

Reply:
Ja ich weiß. Ich fand skillSet in normalen Nachrichten nur hässlich :)